api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +408,25 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	/**
+	 * @see VisitService#getVisitsByPatient(Patient)
+	 */
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyVisitsForNullPatient() {
+		assertEquals(0, visitService.getVisitsByPatient( null).size());
```

Comment:
I suggest you use the following assertion in all your tests for checking if we get an empty list instead of what you used

```java
assertThat(visitService.getVisitsByPatient(null), is(empty()));
```
it will give us a better error message in case the test fails.

You will need to add the following static imports to the top of the test class

```java
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.collection.IsEmptyCollection.empty;
import static org.hamcrest.core.Is.is;
```